packages/notte-sdk/src/notte_sdk/types.py

```
@@ -775,8 +775,12 @@ class SessionStatusResponse(SessionResponse, ReplayResponse):
     pass
 
 
+# Re-export FileInfo from notte_core for use in SDK
+from notte_core.storage import FileInfo as FileInfo  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)